source/_integrations/totalconnect.markdown

```diff
@@ -11,20 +11,12 @@ ha_codeowners:
 ha_domain: totalconnect
 ---
 
-The `totalconnect` integration provides connectivity with the Honeywell TotalConnect alarm systems used by many alarm companies.  
+The `totalconnect` integration provides connectivity with TotalConnect alarm systems used by many alarm companies.  
 
 ## Configuration
 
 To enable TotalConnect via the user interface, go to **Configuration** > **Integrations** > the "plus" button > Total Connect.
 
-To enable TotalConnect via `configuration.yaml` add the following lines:
```

Comment:
I'm confused, we remove YAML config, but we leave in the YAML config block down below?

Reply:
It's removed now.